### DISCUSSION TITLE
can't use additionalConfigFiles

### DISCUSSION THREAD

**Discussion by pbranly** [Lissy93/dashy#97] [Help and Support] [answered]
Hi,
I tried to use that feature but when i Save the config file, i get a :
Validatation Warning: additionalProperties should NOT have additional property: 
what is my mistake ?
thanks
Phil
![2021-07-21 (3)](https://user-images.githubusercontent.com/22018509/126523981-bab71811-cc92-42ba-95da-93e6e1cf6590.png)

**Comment by lissy93** [comment, marked as answer]
Hi @pbranly - Sorry about that, Yup this feature isn't yet merged, as it's not complete. It's something I started working on, but need to implement a better approach, as my current draft isn't quite neat enough. Sorry about that.

Out of interest, where did you find this documented, as I hadn't meant to merge this, as it's still a WIP. 

In the meantime, you can either build from `FEATURE_multi-page-support` (which I wouldn't recommend yet, as there's still a lot that needs to be done to it), or the only other approach is to run multiple containers, and then link them together using the `pageInfo.navLinks` - not ideal, I know, I will build a better approach in the future, but have been quite busy recently, sorry.

**Reply by pbranly** [reply]
Hi
Thank you for quick answer
I found that in the following page: 
https://github.com/Lissy93/dashy/pull/77
as it is written it seems to be implemented 
Don’t worry 
This should be an interesting but not important feature
Congrats for your program
Regards from France
Philippe
